client/go/src/github.com/vespa-engine/vespa/cmd/command_tester.go

```diff
@@ -0,0 +1,46 @@
+// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+// A helper for testing commands
+// Author: bratseth
+
+package cmd
+
+import (
+    "bytes"
+    "github.com/vespa-engine/vespa/utils"
+    "github.com/stretchr/testify/assert"
+	"io/ioutil"
+    "net/http"
+    "testing"
+)
+
+func executeCommand(t *testing.T, args []string) (standardout string) {
+    utils.ActiveHttpClient = mockHttpClient{}
+	b := bytes.NewBufferString("")
+    utils.Out = b
+	rootCmd.SetArgs(args)
+	rootCmd.Execute()
+	out, err := ioutil.ReadAll(b)
+	assert.Empty(t, err, "No error")
+	return string(out)
+}
+
+type mockHttpClient struct {}
```

Comment:
Tip: There's a `httptest` package that makes it easy to start a HTTP server in your test: https://devdocs.io/go/net/http/httptest/index#Server - that way you don't have to mock the client.

Reply:
Thanks! But imho it's better to avoid network stuff in tests unless that's what you are testing.